src/components/navBar/components/DropdownItemsMenu.js

```diff
@@ -25,6 +28,23 @@ const CustomDropdownMenu = ({ classes, handleClick, dropDownElements }) => (
           </NavLink>
         </HashRouter>
       ))}
+
+      {
+        releaseNotesFlag ? (
```

Comment:
Instead of just release notes can we future proof that can create multiple links?
[{title: "releaseNotes", link: "respective link"},{title: "dumm2", link: "respective link2"}]